reactor-netty-core/src/main/java/reactor/netty/tcp/SslProvider.java

```diff
@@ -704,13 +734,15 @@ public final Builder closeNotifyReadTimeoutMillis(long closeNotifyReadTimeoutMil
 		@Override
 		public Builder addSniMapping(String domainName, Consumer<? super SslContextSpec> sslProviderBuilder) {
 			addInternal(domainName, sslProviderBuilder);
+			this.sniMappings = null;
```

Comment:
should we mention in the javadoc that this method and its sisters are exclusive with the `AsyncMapping`-based one?

Reply:
fixed